web-transport-quinn/Cargo.toml

```diff
@@ -11,6 +11,11 @@ edition = "2021"
 keywords = ["quic", "http3", "webtransport"]
 categories = ["network-programming", "web-programming"]
 
+[features]
+default = ["aws-lc-rs"]
+aws-lc-rs = ["dep:aws-lc-rs"]
+ring = ["dep:ring"]
```

Comment:
We should also to enable the feature on rustls/quinn. Otherwise it won't compile without some workarounds.

On the train but I think the syntax is:
```
ring = [ "dep:ring", "quinn/ring", "rustls/ring" ]
```

https://github.com/quinn-rs/quinn/blob/d8302dfaeb7a137fb29d827837b29a613340a7c6/quinn-proto/Cargo.toml#L25